pkgs/development/python-modules/fdroidserver/default.nix

```diff
@@ -1,30 +1,72 @@
-{ stdenv
+{ androguard
+, Babel
 , buildPythonPackage
+, clint
+, defusedxml
+, docker
+, docker-py
 , fetchFromGitLab
+, GitPython
+, isPy3k
 , libcloud
-, pyyaml
+, mwclient
 , paramiko
+, pillow
 , pyasn1
 , pyasn1-modules
-, pillow
-, mwclient
-, GitPython
-, isPy3k
-}:
+, python
+, python-vagrant
+, pyyaml
+, qrcode
+, requests
+, ruamel_yaml
+, stdenv }:
 
 buildPythonPackage rec {
-  version = "2016-05-31";
+  version = "1.1a";
```

Comment:
```suggestion
  version = "1.0.10";
```

Reply:
Actually, using that commit was deliberate. With 1.0.10, I get a weird `AttributeError`. The commit I specified works (with Python 3.7) at least.

Reply:
But `1.1a` makes it look like an official version. You could use `unstable-YYYY-MM-DD` and add a comment why a stable version doesn't work instead. Alternatively, try using Python 3.6.

Reply:
Ah, I agree. I'll add an unstable version.

Reply:
And wouldn't you know it, 1.1 was just released officially :D